### PR TITLE
File words

### PR DESCRIPTION
**Pull request by A-Masoumiyan**

### CODE REVIEW

**Comment by A-Masoumiyan** [comment]
Random Words

File: projects/word-game/words file.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import random
+
+def RandWords(Word_length,Number_of_words):
```
Comment (saeidalidadi) [inline projects/word-game/words file.py]: please use `snake case(snake_case)` convention for variables and  `camel case(camelCase)`  for function names.
@parvinsamin
@somayehe
@aartaa
Also function `arguments` are variables

File: projects/word-game/words file.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+	File = open("words.txt", "r")
+	words = File.read().split()
+	File.close()
+
+	words2=[]
+	for i in words:
+		if len(i) != Word_length:
```
Comment (saeidalidadi) [inline projects/word-game/words file.py]: we don't need any `len` of selected words. just random words with random lenght.
so you should have a function like 
```
selectWords(count=x) # x = number of words from 1 to 10 somehow 
```

Comment (A-Masoumiyan) [reply projects/word-game/words file.py]: متوجه نشدم

Comment (saeidalidadi) [reply projects/word-game/words file.py]: Nemikhad tol kalame ro begiri, mohem nist tolesh cheghade

**Pull request closed without merging**